### DISCUSSION TITLE
Port already in use debugging

### DISCUSSION THREAD

**Discussion by ad3d0** [gameserverapp/Platform#849] [Suggestions]
Would be very convenient if system would also list ports that are detected to be in use.
![Screenshot 2024-05-18 163524](https://github.com/gameserverapp/Platform/assets/43604113/ae1e443a-6813-4b86-9f49-e1ebd5eb20e6)

**Comment by mvaessen** [comment]
Hi, unfortunately is that all the info that is provided by docker. So that's it I'm afraid

**Comment by ad3d0** [comment]
Ohhh, I see. Thank you for reply!
